skyvern/forge/sdk/api/llm/config_registry.py

```diff
@@ -43,6 +46,17 @@ def get_config(cls, llm_key: str) -> LLMRouterConfig | LLMConfig:
         return cls._configs[llm_key]
 
 
+# Check if GPT-4o is enabled and register it
+if SettingsManager.get_settings().ENABLE_GPT_4O:
+    LLMConfigRegistry.register_config(
+        "GPT_4O_MODEL_NAME",  # Replace with the actual model name
+        LLMConfig(
+            "gpt-4o-model-name",  # Replace with the actual model identifier
+            ["GPT_4O_API_KEY"],  # Use the API key defined in your settings
+            supports_vision=False,  # Set according to the model's capabilities
+            add_assistant_prefix=False,  # Adjust based on your naming conventions
+        ),
+    )
```

Comment:
I think the implementation is even simpler than this -- I think you can add a single line below the GPT-4V definition on line 77


    LLMConfigRegistry.register_config(
        "OPENAI_GPT4o", LLMConfig("gpt-4o", ["OPENAI_API_KEY"], supports_vision=True, add_assistant_prefix=False)
    )